src/mod_push_service_mongoosepush.erl

```diff
@@ -125,16 +128,25 @@ http_notification(Host, Method, URL, ReqHeaders, Payload) ->
 %% Helper functions
 %%--------------------------------------------------------------------
 
+parse_api_version("v3") ->
+    {ok, <<"v3">>, v3};
+parse_api_version("v2") ->
+    {ok, <<"v2">>, v2};
+parse_api_version(_) ->
+    {error, not_supported}.
+
 %% Create notification for API v2
-make_notification(v2, Notification, Options = #{<<"silent">> := <<"true">>}) ->
+make_notification(API, Notification, Options = #{<<"silent">> := <<"true">>})
+  when API =:= v2; API =:= v3 ->
```

Comment:
I think this check is overzealous because we can't really have anything other than `v2` and `v3`, right?

Reply:
Currently that's true, the code will fail earlier. Previously we needed to check the version because the atom was created from the config option.